### ISSUE TITLE
Cross Entropy encounters various errors

### ISSUE THREAD

**Issue by stanley-c-yu** [stanley-c-yu/Gradient-Descent-for-Logistic-Regression#1]
Presently the algorithm is unable to properly computer cross-entropy-objective properly every time.  In certain instances, it returns "nan", and often Python reports errors such as but not limited to "overflows" and "log divided by zero".  Currently mired in prep work for a midterm, but I hope to circle back to this by the end of the semester to work these issues out.  

The good news is that the F1 Scores seem to improve as iterations increase, which is an indicator that convergence is occurring.
